sql.scrbl

```diff
@@ -950,7 +949,12 @@ Quasiquotation macro and predicate, respectively, for
                   (@#,lit{unique} column-ident ...)
                   (@#,lit{check} scalar-expr)
                   (@#,lit{foreign-key} column-ident ...
-                     #:references table-ident/columns)]
+                     #:references table-ident/columns
```

Comment:
Make the references and definitions consistent here (referential-action-clause vs action-decl, referential-action vs action). Adding a role prefix to the nonterminal name (eg column-ident vs ident) doesn't seem necessary, and it makes it harder to follow. Remember the reference in the history below.